WooCommerce/Classes/Tools/AppRatings/AppRatingManager.swift

```diff
@@ -305,7 +305,7 @@ public class AppRatingManager {
         static let lastPromptToRateDate = "AppRatingLastPromptToRateDate"
     }
 
-    private enum Constants {
+    enum Constants {
         static let defaultAppReviewURL = URL(string: "https://itunes.apple.com/us/app/id1389130815?mt=8&action=write-review")!
```

Comment:
Do you know if this URL works for all of the different App Stores? I don't have an account in the UK store to test. 😄

Reply:
I tested it in the UK and it works fine for me 🙂